internal/server/deputy_hub.go

```diff
@@ -1,11 +1,12 @@
 package server
 
 import (
-	"github.com/gorilla/mux"
-	"github.com/ministryofjustice/opg-sirius-supervision-deputy-hub/internal/sirius"
 	"net/http"
 	"strconv"
 	"strings"
+
+	"github.com/gorilla/mux"
+	"github.com/ministryofjustice/opg-sirius-supervision-deputy-hub/internal/sirius"
```

Comment:
Could you run the formatter over this? I'm guessing this change is because of a disagreement between the formatter and your IDE.